### PR TITLE
Add check name to constructor

### PR DESCRIPTION
**Pull request by simonmeggle**
As requested, I fixed the PR.

### CODE REVIEW

File: nagiosplugin/check.py [hunk 1]
```diff
@@ -27,19 +27,24 @@
...
 
-    def __init__(self, *objects):
+    def __init__(self, *objects, **kwargs):
         """Creates and configures a check.
 
         Specialized *objects* representing resources, contexts,
         summary, or results are passed to the the :meth:`add` method.
         Alternatively, objects can be added later manually.
+        If no *name* is given, the output prefix is set to the first
+        resource's name. If *name* is None, so prefix is set at all.
```
Comment (mpounsett) [inline nagiosplugin/check.py]: Is this a typo?  ITYM:
```suggestion
        resource's name. If *name* is None, no prefix is set at all.
```

**Review by mpounsett — CHANGES_REQUESTED** [review]

File: nagiosplugin/check.py (hunk 1 above)
Comment (simonmeggle) [reply nagiosplugin/check.py]: (Yes, this is a typo.)

Comment (mpounsett) [reply nagiosplugin/check.py]: OK.  Do you want to push a change to your PR to fix it?  Once that's done I'll merge it in.

Comment (simonmeggle) [reply nagiosplugin/check.py]: I think it's already fixed in 98e786a4, isn't it?

**Review by mpounsett — APPROVED** [review]

**Pull request merged**